script/Ciel/Day 14/14_01B_03_CIEL14B_4_1_2.txt

```diff
@@ -546,8 +546,9 @@ She says the one name she had vowed to never speak again.
 [sha:0d78430f22b0fe01296915bef29aebc7b1db6a36]{
 -- Page 163, Offset 40728.
 -- 志貴にわたしを止めてほしい。
+// On the fence here, kinda want to leave the names for dramatic effect 
```

Comment:
I might keep the names here since I think this is kinda connected to the going insane sequence where she swaps his name with Roa